app/controllers/guides_controller.rb

```diff
@@ -25,17 +25,14 @@ def new
   def create
     @guide = Guide.new(guide_params)
 
-    ActiveRecord::Base.transaction do
-      if @guide.save
-        GuidePublisher.new(guide: @guide).put_draft
-        redirect_to edit_guide_path(@guide), notice: "Guide has been created"
-      else
-        render action: :new
-      end
+    publication = Publisher.new(content_model: @guide).
```

Comment:
I can't remember seeing it in the style guide.. It's a subjective one. I've always chained at the end of the line because when copying and pasting into IRB/console it doesn't handle the dot at the beginning of the line unless it has a trailing `\`. With the dot at the end it knows there is more of the expression coming.. _shrug_ Wadda ya think?

I like pedantic :) I no-one is pedantic then code just ends up an inconsistent mess :+1: